src/android/patches/makeStringsPatch.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function makeStringsPatch(params) {
+  const patch = Object.keys(params).map(key =>
+    `    <string moduleConfig="true" name="${key}">${params[key]}</string>`
```

Comment:
But would you specify something which is not a string or number there? I don't think it would be a case to setup object. I'm not even sure that it's possible to specify something except strings in the plist file.

I just don't want to create a custom file for rnpm, to be honest.